dev/README.md

```diff
@@ -6,3 +6,5 @@ This sub-directory contains the resources related to SCF development.
 | -------- | ---------------------------------------- |
 | minikube | [minikube/README.md](minikube/README.md) |
 | kind     | [kind/README.md](kind/README.md)         |
+
+Also see SCF develop docs in [scf/README.md](scf/README.md).
```

Comment:
Missing `,` after `Also`.

Reply:
Addressed in 23e13e9231af4d88ea31f53bd0888e3fb2bcb86c